docker/arches_her/docker-compose-persist-dependencies.yml

```diff
@@ -0,0 +1,102 @@
+version: '3.8'
+services:
+    nginx-archesher:
+      container_name: nginx-archesher
+      image: nginx:latest
+      ports:
+        - "8080:80"
+      volumes:
+        - ./docker/nginx-config/default.conf:/etc/nginx/conf.d/default.conf
+      networks:
+        - archesher-network
+
+    db-archesher:
+      container_name: db-archesher
+      image: postgis/postgis:14-3.2
+      volumes:
+        - ../../../arches_her-postgres-data:/var/lib/postgresql/data
+        - ../../../arches_her-postgres-log:/var/log/postgresql
+        - ./docker/init-unix.sql:/docker-entrypoint-initdb.d/init.sql # to set up the DB template
+      ports:
+        - '5433:5432'
+      env_file: 
+        - ./docker/env_file.env
+      networks:
+        - archesher-network
+
+    elastic-bindmount-init-archesher:
+      image: alpine
+      container_name: elastic-bindmount-init-archesher
+      restart: "no"
+      entrypoint: |
+        /bin/sh -c "chown 1000:1000 /esBindMount"
+      volumes:
+        - ../../../arches_her-elasticsearch-data:/esBindMount
+
+
+    elasticsearch-archesher:
+      container_name: elasticsearch-archesher
+      image: elasticsearch:8.5.0
```

Comment:
@RobOatesHistoricEngland - this should match the ES 8.10.4 version in the other dependancy compose

Reply:
Done